### DISCUSSION TITLE
ubuntu 24.04

### DISCUSSION THREAD

**Discussion by lvazdela** [abraunegg/onedrive#2867] [Q&A] [answered]
Hi,
Does onedrive v 2.5.2 works with ubuntu 24.04 LTS?
Thanks

**Comment by abraunegg** [comment, marked as answer]
Please read the documentation to answer this question

**Reply by lvazdela** [reply]
The documentation may be a little confusing:
1.- In [https://github.com/abraunegg/onedrive/blob/master/docs/ubuntu-package-install.md](https://github.com/abraunegg/onedrive/discussions/url) the directions to download onedrive for Ubuntu 24.04 from opensuse are straight forward, but it doesn´t state what version of onedrive is going to be downloaded. only after installing onedrive you know that is the version 2.5.2
2.- In contrast, in [https://github.com/abraunegg/onedrive/blob/master/docs/install.md](https://github.com/abraunegg/onedrive/discussions/url) (a link that appears in [https://github.com/abraunegg/onedrive/blob/master/docs/install.md](https://github.com/abraunegg/onedrive/discussions/url)) the table advice to use onedrive 2.4.25 for ubuntu 24.04.

**Reply by abraunegg** [reply]
The table explicitly advises to *not* use Ubuntu Universe packages and advises you to use the OpenSuSE Build Service packages.

Please re read the page.
